### DISCUSSION TITLE
I can't get student account

### DISCUSSION THREAD

**Discussion by FZT1999** [community/community#36321] [GitHub Education] [answered]
Hi i cant get student account and it doesn't let me use the back camera and sends me error and give error your university is not recognize , I'm student from university of Isfahan

**Comment by ettaboyle** [comment, marked as answer]
Hi @FZT1999 Thanks for reaching out! You can find all the details you need about the GitHub Student Developer Pack in this post:
[Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)

Please review all the information in that post and follow the steps outlined. Thank you!

**Reply by adii800** [reply]
Hi @ettaboyle - I believe this student was referring to the back camera not being recognized when attempting to upload documentation via mobile. I encountered the same issue as well (iPhone XS, browsers: Chrome, Safari).

**Comment by AdnanTanoli90** [comment]
hi I had a student account but it was with my old account which i have no access of now i want to get my student email unlink from it so i can again applu for student account on my new account please
